common/reviews/api/rush-lib.api.md

```diff
@@ -539,7 +539,7 @@ export class RushConfiguration {
     get repositoryDefaultBranch(): string;
     get repositoryDefaultFullyQualifiedRemoteBranch(): string;
     get repositoryDefaultRemote(): string;
-    get repositoryUrl(): string | undefined;
+    get repositoryUrls(): string[] | undefined;
```

Comment:
This is technically a breaking API change.  However it's very unlikely that any scripts are actually relying on it, so I'm fine with skipping the `@deprecated` property, as long as we call this out in the CHANGELOG.

Reply:
Yeah it's called out in the changelog.